src/app/(proper_react)/redesign/(public)/page.tsx

```diff
@@ -14,11 +18,18 @@ export default async function Page() {
   const countryCode = getCountryCode(headers());
   const eligibleForPremium = isEligibleForPremium(countryCode, enabledFlags);
 
+  // request the profile stats for the last 30 days
+  const profileStats = await getProfilesStats(
+    new Date(new Date().setDate(new Date().getDate() - 30)),
```

Comment:
Wondering if there is an easier way to do this without 3 Date constructors.
We could just create one more temp date variable and modify that:
```js
const now = new Date();
now.setDate(now.getDate() - 30);
console.log(now.toLocaleDateString()); // "12/3/2023"
```

Or even `Date.now()` and subtract 30 [days] * 24 [hours] * 60 [minutes] * 60 [seconds] * 1000 [milliseconds]:
```sh
const now2 = new Date(Date.now() - (30 * 24 * 60 * 60 * 1000));
console.log(now2.toLocaleDateString()); // "12/3/2023"
```

Reply:
thanks fixed